src/nvim/normal.c

```diff
@@ -2106,42 +2106,44 @@ int get_vtopline(win_T *wp)
 /// scrolled.  Called from normal_cmd() and edit().
 void do_check_scrollbind(bool check)
 {
+  if (!check || !curwin->w_p_scb) {
+    return;
```

Comment:
The static variables still need to be updated.

Reply:
Yes, just realised.